src/envWrapper.ts

```diff
@@ -0,0 +1,4 @@
+//simple function to replace an external dependency, will be swapped for some other method
+export function getEnv(lookup: string): string {
```

Comment:
This [used to throw](https://github.com/therockstorm/utils/blob/master/src/index.ts#L19). How does the failure state look if it can't get these values vs if it throws immediately?

Reply:
Thought about this a bit while writing it, it may make debugging slightly more difficult without the throw should an environment variable not be set (just present as an error calling bamboo or slack vs failing right away), but I don't think we've seen issues there, thus I opted for simplicity when doing the first draft. I'm also assuming we will be moving from the env variables here in the next few weeks. Open to changing however if we think its worth it.